Circle_calculator.html

```diff
@@ -0,0 +1,40 @@
+<!-- <!DOCTYPE html> -->
+<html lang="en">
+<head>
+    <meta charset='utf-8'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+</head>
+<body>
+
+</body>
+    <script>
+
+        class Circle{
+            constructor(radius){
+                this.radius = radius;
+            }
+            getDiameter(){
+                // console.log('getDiameter');
+                return console.log(`Diameter is : ${this.radius*2}`);
+            }
+            getCircumference(){
+                // console.log('getCircumference');
+                return console.log(`Circumference is : ${2* Math.PI *this.radius}`);
+            }
+            getArea(){
+                // console.log('getArea');
+                return console.log(`Area is : ${Math.PI * this.radius * this.radius}`);
+            }
+
+        }
+        
+        const circle = new Circle(10);
+        circle.getDiameter(); // 20
+        circle.getCircumference(); // 62.83
+        circle.getArea(); // 314.15

```

Comment:
The implementation looks correct though there are 2 things to note here:

- This should have been in a file called `warmup.js` and not inside of a HTML page. With Node.js we run JavaScript on the server (which in this case is your laptop), while frontend JavaScript runs in the browser.
- The other thing is that `getDiameter` and the 2 other functions should *return* numbers, it should not log the results. Logging to the console is still a good idea, but you should do that outside of the class.

Let me know if this isn't clear.